charts/backstage/values.yaml

```diff
@@ -59,8 +59,11 @@ upstream:
             password: ${POSTGRESQL_ADMIN_PASSWORD}
             user: postgres
         auth:
-          keys:
-            - secret: ${BACKEND_SECRET}
+          externalAccess:
+            - type: legacy
+              options:
+                subject: legacy-default-cofig
```

Comment:
Is this a typo? Should it be `legacy-default-config`?

Reply:
Oh yes, that's a typo